src/jquery.valueview.valueview.js

```diff
@@ -78,8 +78,7 @@ function expertProxy( fnName ) {
  *        Time milliseconds that the parser should wait before parsing. A delay is useful to limit
  *        the number of API request that are outdated when returning because the input has changed
  *        in the meantime.
- * @param {Object|null} [options.mediaWiki=null]
- *        `mediaWiki` JavaScript object that may be used when in MediaWiki environment.
+ * @param {util.MessageProvider|null} [options.messageProvider=null]
```

Comment:
Please add some option description. What is the option used for and/or why would it make sense to specify the option?